doc/article/en-US/angular1-to-aurelia.md

```diff
@@ -0,0 +1,526 @@
+
+Instead of a configuration object, Aurelia takes a class instance.
+
+An Angular directive may have looked like this:
+
+<code-listing heading="Angular Custom Components">
+	<source-code lang="ES 2015/2016">
+		angular.directive('display', ['$window', '$document', 'animate', function ($window, $document, animate) {
+			return {
+				scope: true,
+				restrict: 'A',
+				templateUrl: 'myTemplate.html',
+				link: function (scope, element, attrs) {
+					var windowheight = angular.element($window).height();
+		
+					scope.holeHeight = function (depth) {
+						return (depth * 0.2) < windowheight ? (depth * 0.2) + 'px' : undefined;
+					};
+		
+					scope.bgPos = function (depth) {
+						return (depth * 0.2) < windowheight ? 0 : (depth - windowheight) * 0.2;
+					};
+		
+					scope.holeWidth = function () {
+						return Math.min(700, Math.max(350, _(scope.shop)
+							.reduce(function (p, c) {
+								return p + c.owned * c.digValue;
+							}, 0)
+						));
+					};
+		
+					scope.displayArray = [];
+		
+					scope.$watch('shop', function () {
+						scope.displayArray = _.values(scope.shop).reverse();
+					});
+		
+					(function tick(timestamp) {
+						angular.element($window).scrollTop($document.height());
+						animate(tick);
+					})();
+				}
+			};
+		}])
+	</source-code>
+</code-listing>
+
+Converting this directive to an Aurelia custom element requires us to remove the scope references, migrate from `link` and `controller` properties to lifecycle methods, and simplify our object. Finally, we'll have to use Aurelia's decorator functions to register our object and its configuration as a custom element instance.
+
+<code-listing heading="Aurelia Custom Components">
+	<source-code lang="ES 2015/2016">		export class Display {
+			constructor (window, document, animate) {
+				this.window = window;
+				this.document = document;
+				this.animate = animate;
+			}
+			
+			attached () {
+
+			}
+		}
+	</source-code>
+</code-listing>
+
+> Warning: Scope
+> Custom elements in Aurelia cannot access the outer scope. This means that all custom elements in Aurelia essentially behave as though they had the Angular property `scope: { ... }` - custom elements have an isolated scope.
+
+#### Converting an Angular directive to an Aurelia custom attribute
+
+Custom attributes are handled similarly to custom elements in Aurelia. Custom attributes also use an isolate scope - they cannot access the outer scope.
+
+#### Transclusion
+
+Aurelia provides the `content` tag to provide transclusion features. This is part of the view itself, instead of being part of the directive.
+
+####
+
+## [Quick Reference](aurelia-doc://section/4/version/1.0.0)
+
+#### $scope
+
+No equivalent exists in Aurelia.
+
+Instead of `$scope.property`, use `this.property` inside of a viewmodel object.
+
+Instead of `$scope.$watch()`, use a computed property - or, to trigger a function, use the BindingEngine.
```

Comment:
> or, to trigger a function, use the BindingEngine

With my experience, this is rarely needed or reasonable (unless writing plugins, where this is much more often handy). It might be worth to explain how to get rid of `$scope.$watch` that is used in Angular app for different purposes (mostly as a workaround?).